modules/transformations/redshift/doc/ST_DESTINATION.md

```diff
@@ -0,0 +1,25 @@
+### ST_DESTINATION
+
+{{% bannerNote type="code" %}}
+transformations.ST_DESTINATION(geog, distance, bearing, units)
+{{%/ bannerNote %}}
+
+**Description**
+
+Takes a Point and calculates the location of a destination point given a distance in degrees, radians, miles, or kilometers; and bearing in degrees. This uses the Haversine formula to account for global curvature.
+
+* `geog`: `GEOMETRY` starting point.
+* `distance`: `FLOAT8` distance from the origin point.
+* `bearing`: `FLOAT8` ranging from -180 to 180.
+* `units` (optional): `VARCHAR(15)` units of length, the supported options are: miles, kilometers, degrees or radians. If `NULL`the default value `kilometers` is used.
```

Comment:
As redshift supports function overloading we should do the same that in SF. having the two versions. Also the two examples and avoid passing `NULL`.